deepchecks/vision/checks/distribution/new_labels.py

```diff
@@ -0,0 +1,204 @@
+    if data.task_type == TaskType.OBJECT_DETECTION:
+        formatted_labels_of_image = data.batch_to_labels(batch)[0].numpy()
+        bboxes_of_class_id = torch.tensor([x for x in formatted_labels_of_image if x[0] == class_id])
+        image = draw_bboxes(image, bboxes_of_class_id, copy_image=False, border_width=5)
+
+    image_thumbnail = prepare_thumbnail(
+        image=image,
+        size=THUMBNAIL_SIZE,
+        copy_image=False
+    )
+    return image_thumbnail
+
+
+class NewLabels(TrainTestCheck):
+    """Detects labels that apper only in the test set.
+
+    Parameters
+    ----------
+    max_images_to_display_per_label : int , default: 3
+        maximum number of images to show from each newly found label in the test set.
+    max_new_labels_to_display : int , default: 3
+        Maximum number of new labels to display in output.
+    """
+
+    def __init__(
+            self,
+            max_images_to_display_per_label: int = 3,
+            max_new_labels_to_display: int = 3,
+            **kwargs
+    ):
+        super().__init__(**kwargs)
+        # validate input parameters:
+        if not isinstance(max_images_to_display_per_label, int):
+            raise DeepchecksValueError('max_num_images_to_display_per_label must be an integer')
+        if not isinstance(max_new_labels_to_display, int):
+            raise DeepchecksValueError('max_num_new_labels_to_display must be an integer')
+
+        self.max_images_to_display_per_label = max_images_to_display_per_label
+        self.max_new_labels_to_display = max_new_labels_to_display
+        self._class_id_counter = defaultdict(list)
+
+    def update(self, context: Context, batch: Batch, dataset_kind):
+        """Count number of appearances for each class_id in train and test."""
+        data = context.get_data_by_kind(dataset_kind)
+        classes_in_batch = data.get_classes(batch.labels)
+        for labels_per_image in classes_in_batch:
+            for label in labels_per_image:
+                if label not in self._class_id_counter:
+                    self._class_id_counter[label] = [0, 0]
+                self._class_id_counter[label][dataset_kind == DatasetKind.TEST] += 1
+
+    def compute(self, context: Context) -> CheckResult:
+        """Calculate which class_id are only available in the test data set and display them.
+
+        Returns
+        -------
+        CheckResult
+            value: A dictionary showing new class_ids introduced in the test set and number of times they were spotted.
+            display: Images containing said class_ids from the test set.
+        """
+        data = context.get_data_by_kind(DatasetKind.TEST)
+        class_id_only_in_test = defaultdict(list)
+        for class_id, value in self._class_id_counter.items():
+            if value[0] == 0:
+                class_id_only_in_test[class_id] = value[1]
+        class_id_only_in_test = dict(sorted(class_id_only_in_test.items(), key=lambda item: -item[1]))
```

Comment:
Add a comment here explaining that your sorting by number of occurrences in test in descending order